src/EhrgoHealth.Web/Startup.cs

```diff
@@ -24,56 +28,85 @@ public void Configuration(IAppBuilder app)
             builder.Register((a) =>
             {
                 var context = a.Resolve<HttpContextBase>();
-                if(context != null)
+                if (context != null)
                 {
                     return context.GetOwinContext();
                 }
                 return null;
-            });
+            }).InstancePerRequest();
             builder.Register((a) =>
             {
                 var context = a.Resolve<IOwinContext>();
-                if(context != null)
+                if (context != null)
                 {
                     return context.Get<ApplicationSignInManager>();
                 }
                 return null;
-            });
+            }).InstancePerRequest();
             builder.Register((a) =>
             {
                 var context = a.Resolve<IOwinContext>();
-                if(context != null)
+                if (context != null)
                 {
                     return context.Get<ApplicationUserManager>();
                 }
                 return null;
-            });
+            }).InstancePerRequest();
             builder.Register((a) =>
             {
                 var context = a.Resolve<IOwinContext>();
-                if(context != null)
+                if (context != null)
                 {
                     return context.Get<ApplicationRoleManager>();
                 }
                 return null;
-            });
+            }).InstancePerRequest();
 
             builder.Register((a) =>
             {
                 var context = a.Resolve<IOwinContext>();
-                if(context != null)
+                if (context != null)
                 {
                     return context.Authentication;
                 }
                 return null;
+            }).InstancePerRequest();
+            builder.Register(a =>
+            {
+                var options = new FitbitAuthenticationOptions()
+                {
+                    //probably shouldnt bake api keys here, but for now it works
+                    ClientId = "227PBF",
+                    ClientSecret = "dab2395907e7bec0317723bd2f13f4d1"
```

Comment:
it should still work as expected (everyones data gets imported properly) they are just authentication keys for fitbit, so anyone whom reads this github could theoretically make requests  as our application.